docs/advanced-usage/custom-settings.md

```diff
@@ -12,9 +14,10 @@ void setup() {
 }
 ```
 
-!!! tip "setDefaultValue() before Homie.setup()"
-    As shown in the example above, the **default value** has to be set **before** `Homie.setup()` is called.
-    Otherwise you get an error on startup if there is also no value configured in JSON configuration file.
+**Tip:** "setDefaultValue() before Homie.setup()"
```

Comment:
Please let the `!!! ` to be formatted on the webpage as info box:

![image](https://user-images.githubusercontent.com/184547/74775821-c1e7fe00-5296-11ea-8578-73224b0942d1.png)

Reply:
```suggestion
!!! tip "setDefaultValue() before Homie.setup()"
```